go.mod

```diff
@@ -79,6 +79,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver v0.40.0
```

Comment:
mind moving up w/ the other receivers?

Reply:
Ack, I installed it with `go mod tidy` and didn't check it. Thanks.